config.go

```diff
@@ -309,6 +309,8 @@ type Config struct {
 
 	Routing *lncfg.Routing `group:"routing" namespace:"routing"`
 
+	Gossip *lncfg.Gossip `group:"gossip" namespace:"gossip"`
```

Comment:
Thoughts on moving the existing `numgraphsyncpeers` flag here and deprecating the top-level one while still parsing it? Then we can remove the top-level one at a later release.

Reply:
sgtm, let's do that in a follow up